client/components/Project/ProjectComponent.vue

```diff
@@ -50,19 +80,48 @@ export default {
           id: this.project._id,
         },
       });
+    },
+    markDelete() {
+      // TODO(AL): call backend when ready
+      this.showEdit = false;
+    },
+    async toggleActive() {
+      const options = {
+        method: 'PATCH',
+        headers: {'Content-Type': 'application/json'},
+        credentials: 'same-origin',
```

Comment:
unsure if this is part of the todo but you need the body here with active=false right?